paddle/fluid/inference/api/paddle_analysis_config.h

```diff
@@ -34,26 +34,69 @@ class AnalysisPredictor;
 namespace contrib {
 
 // NOTE WIP, not stable yet.
```

Comment:
Should this comment be deleted?

Reply:
Some following modification will be made, and will delete this before v1.3 release.